<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="17" ignored="0" total="55" passed="38">
  <reporter-output>
  </reporter-output>
  <suite name="Demo Suite" duration-ms="3016" started-at="2019-08-19T15:51:37 GMT+04:00" finished-at="2019-08-19T15:51:41 GMT+04:00">
    <groups>
    </groups>
    <test name="Sub Cal Test" duration-ms="79" started-at="2019-08-19T15:51:37 GMT+04:00" finished-at="2019-08-19T15:51:38 GMT+04:00">
      <class name="com.epam.cdp.tests.SubCalTest">
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.epam.cdp.tests.SubCalTest@194a1b5]" name="setUp" is-config="true" duration-ms="31" started-at="2019-08-19T15:51:38 GMT+04:00" finished-at="2019-08-19T15:51:38 GMT+04:00">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="twoMinusOne(long, long, long)[pri:0, instance:com.epam.cdp.tests.SubCalTest@194a1b5]" name="twoMinusOne" duration-ms="0" started-at="2019-08-19T15:51:38 GMT+04:00" data-provider="valuesForSubTest" finished-at="2019-08-19T15:51:38 GMT+04:00">
          <params>
            <param index="0">
              <value>
                <![CDATA[10]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[23]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[-13]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- twoMinusOne -->
        <test-method status="PASS" signature="twoMinusOne(long, long, long)[pri:0, instance:com.epam.cdp.tests.SubCalTest@194a1b5]" name="twoMinusOne" duration-ms="0" started-at="2019-08-19T15:51:38 GMT+04:00" data-provider="valuesForSubTest" finished-at="2019-08-19T15:51:38 GMT+04:00">
          <params>
            <param index="0">
              <value>
                <![CDATA[10]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[-1]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[11]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- twoMinusOne -->
        <test-method status="PASS" signature="twoMinusOne(long, long, long)[pri:0, instance:com.epam.cdp.tests.SubCalTest@194a1b5]" name="twoMinusOne" duration-ms="0" started-at="2019-08-19T15:51:38 GMT+04:00" data-provider="valuesForSubTest" finished-at="2019-08-19T15:51:38 GMT+04:00">
          <params>
            <param index="0">
              <value>
                <![CDATA[21]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[10]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[11]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- twoMinusOne -->
        <test-method status="PASS" signature="twoMinusOne(long, long, long)[pri:0, instance:com.epam.cdp.tests.SubCalTest@194a1b5]" name="twoMinusOne" duration-ms="0" started-at="2019-08-19T15:51:38 GMT+04:00" data-provider="valuesForSubTest" finished-at="2019-08-19T15:51:38 GMT+04:00">
          <params>
            <param index="0">
              <value>
                <![CDATA[0]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[0]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[0]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- twoMinusOne -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.epam.cdp.tests.SubCalTest@194a1b5]" name="tearDown" is-config="true" duration-ms="0" started-at="2019-08-19T15:51:38 GMT+04:00" finished-at="2019-08-19T15:51:38 GMT+04:00">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.epam.cdp.tests.SubCalTest -->
    </test> <!-- Sub Cal Test -->
    <test name="Sum Cal Test" duration-ms="0" started-at="2019-08-19T15:51:38 GMT+04:00" finished-at="2019-08-19T15:51:38 GMT+04:00">
      <class name="com.epam.cdp.tests.SumCalTest">
        <test-method status="PASS" signature="onePlusTwo(long, long, long)[pri:0, instance:com.epam.cdp.tests.SumCalTest@1f983a6]" name="onePlusTwo" duration-ms="0" started-at="2019-08-19T15:51:38 GMT+04:00" data-provider="valuesForSumTest" finished-at="2019-08-19T15:51:38 GMT+04:00">
          <params>
            <param index="0">
              <value>
                <![CDATA[-1]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[10]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[9]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- onePlusTwo -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.epam.cdp.tests.SumCalTest@1f983a6]" name="tearDown" is-config="true" duration-ms="0" started-at="2019-08-19T15:51:38 GMT+04:00" finished-at="2019-08-19T15:51:38 GMT+04:00">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="onePlusTwo(long, long, long)[pri:0, instance:com.epam.cdp.tests.SumCalTest@1f983a6]" name="onePlusTwo" duration-ms="0" started-at="2019-08-19T15:51:38 GMT+04:00" data-provider="valuesForSumTest" finished-at="2019-08-19T15:51:38 GMT+04:00">
          <params>
            <param index="0">
              <value>
                <![CDATA[0]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[0]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[0]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- onePlusTwo -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.epam.cdp.tests.SumCalTest@1f983a6]" name="setUp" is-config="true" duration-ms="0" started-at="2019-08-19T15:51:38 GMT+04:00" finished-at="2019-08-19T15:51:38 GMT+04:00">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="onePlusTwo(long, long, long)[pri:0, instance:com.epam.cdp.tests.SumCalTest@1f983a6]" name="onePlusTwo" duration-ms="0" started-at="2019-08-19T15:51:38 GMT+04:00" data-provider="valuesForSumTest" finished-at="2019-08-19T15:51:38 GMT+04:00">
          <params>
            <param index="0">
              <value>
                <![CDATA[10]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[23]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[33]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- onePlusTwo -->
        <test-method status="PASS" signature="onePlusTwo(long, long, long)[pri:0, instance:com.epam.cdp.tests.SumCalTest@1f983a6]" name="onePlusTwo" duration-ms="0" started-at="2019-08-19T15:51:38 GMT+04:00" data-provider="valuesForSumTest" finished-at="2019-08-19T15:51:38 GMT+04:00">
          <params>
            <param index="0">
              <value>
                <![CDATA[10]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[-1]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[9]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- onePlusTwo -->
      </class> <!-- com.epam.cdp.tests.SumCalTest -->
    </test> <!-- Sum Cal Test -->
    <test name="Mult Cal Test" duration-ms="15" started-at="2019-08-19T15:51:38 GMT+04:00" finished-at="2019-08-19T15:51:38 GMT+04:00">
      <class name="com.epam.cdp.tests.MultCalTest">
        <test-method status="PASS" signature="oneMultTwoTest(long, long, long)[pri:0, instance:com.epam.cdp.tests.MultCalTest@1f256fa]" name="oneMultTwoTest" duration-ms="0" started-at="2019-08-19T15:51:38 GMT+04:00" data-provider="valuesForMultTest" finished-at="2019-08-19T15:51:38 GMT+04:00">
          <params>
            <param index="0">
              <value>
                <![CDATA[0]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[0]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[0]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- oneMultTwoTest -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.epam.cdp.tests.MultCalTest@1f256fa]" name="setUp" is-config="true" duration-ms="0" started-at="2019-08-19T15:51:38 GMT+04:00" finished-at="2019-08-19T15:51:38 GMT+04:00">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="doubleOneMultTwoTest(double, double, double)[pri:0, instance:com.epam.cdp.tests.MultCalTest@1f256fa]" name="doubleOneMultTwoTest" duration-ms="0" started-at="2019-08-19T15:51:38 GMT+04:00" data-provider="doubleValuesForMultTest" finished-at="2019-08-19T15:51:38 GMT+04:00">
          <params>
            <param index="0">
              <value>
                <![CDATA[-1.55]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[-100]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[155.0]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- doubleOneMultTwoTest -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.epam.cdp.tests.MultCalTest@1f256fa]" name="tearDown" is-config="true" duration-ms="0" started-at="2019-08-19T15:51:38 GMT+04:00" finished-at="2019-08-19T15:51:38 GMT+04:00">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="oneMultTwoTest(long, long, long)[pri:0, instance:com.epam.cdp.tests.MultCalTest@1f256fa]" name="oneMultTwoTest" duration-ms="0" started-at="2019-08-19T15:51:38 GMT+04:00" data-provider="valuesForMultTest" finished-at="2019-08-19T15:51:38 GMT+04:00">
          <params>
            <param index="0">
              <value>
                <![CDATA[6]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[10]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[60]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- oneMultTwoTest -->
        <test-method status="PASS" signature="oneMultTwoTest(long, long, long)[pri:0, instance:com.epam.cdp.tests.MultCalTest@1f256fa]" name="oneMultTwoTest" duration-ms="0" started-at="2019-08-19T15:51:38 GMT+04:00" data-provider="valuesForMultTest" finished-at="2019-08-19T15:51:38 GMT+04:00">
          <params>
            <param index="0">
              <value>
                <![CDATA[10]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[-1]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[-10]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- oneMultTwoTest -->
        <test-method status="PASS" signature="oneMultTwoTest(long, long, long)[pri:0, instance:com.epam.cdp.tests.MultCalTest@1f256fa]" name="oneMultTwoTest" duration-ms="0" started-at="2019-08-19T15:51:38 GMT+04:00" data-provider="valuesForMultTest" finished-at="2019-08-19T15:51:38 GMT+04:00">
          <params>
            <param index="0">
              <value>
                <![CDATA[5]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[3]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[15]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- oneMultTwoTest -->
        <test-method status="PASS" signature="doubleOneMultTwoTest(double, double, double)[pri:0, instance:com.epam.cdp.tests.MultCalTest@1f256fa]" name="doubleOneMultTwoTest" duration-ms="0" started-at="2019-08-19T15:51:38 GMT+04:00" data-provider="doubleValuesForMultTest" finished-at="2019-08-19T15:51:38 GMT+04:00">
          <params>
            <param index="0">
              <value>
                <![CDATA[10]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[-1.5]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[-15.0]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- doubleOneMultTwoTest -->
        <test-method status="FAIL" signature="doubleOneMultTwoTest(double, double, double)[pri:0, instance:com.epam.cdp.tests.MultCalTest@1f256fa]" name="doubleOneMultTwoTest" duration-ms="0" started-at="2019-08-19T15:51:38 GMT+04:00" data-provider="doubleValuesForMultTest" finished-at="2019-08-19T15:51:38 GMT+04:00">
          <params>
            <param index="0">
              <value>
                <![CDATA[1.5]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[3]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[4.5]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Invalid result of mult operation! did not expect to find [4.5] but found [4.0]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Invalid result of mult operation! did not expect to find [4.5] but found [4.0]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.failNotEquals(Assert.java:969)
at org.testng.Assert.assertEquals(Assert.java:640)
at com.epam.cdp.tests.MultCalTest.doubleOneMultTwoTest(MultCalTest.java:26)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1175)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at com.epam.cdp.tests.testng.Runner.main(Runner.java:24)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- doubleOneMultTwoTest -->
        <test-method status="PASS" signature="doubleOneMultTwoTest(double, double, double)[pri:0, instance:com.epam.cdp.tests.MultCalTest@1f256fa]" name="doubleOneMultTwoTest" duration-ms="0" started-at="2019-08-19T15:51:38 GMT+04:00" data-provider="doubleValuesForMultTest" finished-at="2019-08-19T15:51:38 GMT+04:00">
          <params>
            <param index="0">
              <value>
                <![CDATA[0.0]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[0.0]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[0.0]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- doubleOneMultTwoTest -->
      </class> <!-- com.epam.cdp.tests.MultCalTest -->
    </test> <!-- Mult Cal Test -->
    <test name="Div Cal Test" duration-ms="0" started-at="2019-08-19T15:51:39 GMT+04:00" finished-at="2019-08-19T15:51:39 GMT+04:00">
      <class name="com.epam.cdp.tests.DivCalTest">
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.epam.cdp.tests.DivCalTest@18c7aca]" name="setUp" is-config="true" duration-ms="0" started-at="2019-08-19T15:51:39 GMT+04:00" finished-at="2019-08-19T15:51:39 GMT+04:00">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="doubleOneDivTwoTest(double, double, double)[pri:0, instance:com.epam.cdp.tests.DivCalTest@18c7aca]" name="doubleOneDivTwoTest" duration-ms="0" started-at="2019-08-19T15:51:39 GMT+04:00" data-provider="doubleValuesForDivTest" finished-at="2019-08-19T15:51:39 GMT+04:00">
          <params>
            <param index="0">
              <value>
                <![CDATA[1.5]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[3]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[0.5]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- doubleOneDivTwoTest -->
        <test-method status="PASS" signature="doubleOneDivTwoTest(double, double, double)[pri:0, instance:com.epam.cdp.tests.DivCalTest@18c7aca]" name="doubleOneDivTwoTest" duration-ms="0" started-at="2019-08-19T15:51:39 GMT+04:00" data-provider="doubleValuesForDivTest" finished-at="2019-08-19T15:51:39 GMT+04:00">
          <params>
            <param index="0">
              <value>
                <![CDATA[0.0]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[1]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[0.0]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- doubleOneDivTwoTest -->
        <test-method status="PASS" signature="doubleOneDivTwoTest(double, double, double)[pri:0, instance:com.epam.cdp.tests.DivCalTest@18c7aca]" name="doubleOneDivTwoTest" duration-ms="0" started-at="2019-08-19T15:51:39 GMT+04:00" data-provider="doubleValuesForDivTest" finished-at="2019-08-19T15:51:39 GMT+04:00">
          <params>
            <param index="0">
              <value>
                <![CDATA[-1.55]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[-1.55]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[1.0]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- doubleOneDivTwoTest -->
        <test-method status="PASS" signature="oneDivTwoTest(long, long, long)[pri:0, instance:com.epam.cdp.tests.DivCalTest@18c7aca]" name="oneDivTwoTest" duration-ms="0" started-at="2019-08-19T15:51:39 GMT+04:00" data-provider="valuesForDivTest" finished-at="2019-08-19T15:51:39 GMT+04:00">
          <params>
            <param index="0">
              <value>
                <![CDATA[0]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[1]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[0]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- oneDivTwoTest -->
        <test-method status="PASS" signature="doubleOneDivTwoTest(double, double, double)[pri:0, instance:com.epam.cdp.tests.DivCalTest@18c7aca]" name="doubleOneDivTwoTest" duration-ms="0" started-at="2019-08-19T15:51:39 GMT+04:00" data-provider="doubleValuesForDivTest" finished-at="2019-08-19T15:51:39 GMT+04:00">
          <params>
            <param index="0">
              <value>
                <![CDATA[3]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[-1.5]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[-2.0]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- doubleOneDivTwoTest -->
        <test-method status="PASS" signature="oneDivTwoTest(long, long, long)[pri:0, instance:com.epam.cdp.tests.DivCalTest@18c7aca]" name="oneDivTwoTest" duration-ms="0" started-at="2019-08-19T15:51:39 GMT+04:00" data-provider="valuesForDivTest" finished-at="2019-08-19T15:51:39 GMT+04:00">
          <params>
            <param index="0">
              <value>
                <![CDATA[-10]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[-10]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[1]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- oneDivTwoTest -->
        <test-method status="PASS" signature="oneDivTwoTest(long, long, long)[pri:0, instance:com.epam.cdp.tests.DivCalTest@18c7aca]" name="oneDivTwoTest" duration-ms="0" started-at="2019-08-19T15:51:39 GMT+04:00" data-provider="valuesForDivTest" finished-at="2019-08-19T15:51:39 GMT+04:00">
          <params>
            <param index="0">
              <value>
                <![CDATA[6]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[3]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[2]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- oneDivTwoTest -->
        <test-method status="PASS" signature="oneDivTwoTest(long, long, long)[pri:0, instance:com.epam.cdp.tests.DivCalTest@18c7aca]" name="oneDivTwoTest" duration-ms="0" started-at="2019-08-19T15:51:39 GMT+04:00" data-provider="valuesForDivTest" finished-at="2019-08-19T15:51:39 GMT+04:00">
          <params>
            <param index="0">
              <value>
                <![CDATA[10]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[-1]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[-10]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- oneDivTwoTest -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.epam.cdp.tests.DivCalTest@18c7aca]" name="tearDown" is-config="true" duration-ms="0" started-at="2019-08-19T15:51:39 GMT+04:00" finished-at="2019-08-19T15:51:39 GMT+04:00">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.epam.cdp.tests.DivCalTest -->
    </test> <!-- Div Cal Test -->
    <test name="Cos Cal Test" duration-ms="0" started-at="2019-08-19T15:51:39 GMT+04:00" finished-at="2019-08-19T15:51:39 GMT+04:00">
      <class name="com.epam.cdp.tests.CosCalTest">
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.epam.cdp.tests.CosCalTest@1b6a447]" name="tearDown" is-config="true" duration-ms="0" started-at="2019-08-19T15:51:39 GMT+04:00" finished-at="2019-08-19T15:51:39 GMT+04:00">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="FAIL" signature="CosTest(double, double)[pri:0, instance:com.epam.cdp.tests.CosCalTest@1b6a447]" name="CosTest" duration-ms="0" started-at="2019-08-19T15:51:39 GMT+04:00" data-provider="valuesForCosTest" finished-at="2019-08-19T15:51:39 GMT+04:00">
          <params>
            <param index="0">
              <value>
                <![CDATA[90]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[0]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Invalid result of cos operation! did not expect to find [0.0] but found [0.8939966636005579]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Invalid result of cos operation! did not expect to find [0.0] but found [0.8939966636005579]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.failNotEquals(Assert.java:969)
at org.testng.Assert.assertEquals(Assert.java:640)
at com.epam.cdp.tests.CosCalTest.CosTest(CosCalTest.java:11)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1175)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at com.epam.cdp.tests.testng.Runner.main(Runner.java:24)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- CosTest -->
        <test-method status="FAIL" signature="CosTest(double, double)[pri:0, instance:com.epam.cdp.tests.CosCalTest@1b6a447]" name="CosTest" duration-ms="0" started-at="2019-08-19T15:51:39 GMT+04:00" data-provider="valuesForCosTest" finished-at="2019-08-19T15:51:39 GMT+04:00">
          <params>
            <param index="0">
              <value>
                <![CDATA[0]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[1]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Invalid result of cos operation! did not expect to find [1.0] but found [0.0]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Invalid result of cos operation! did not expect to find [1.0] but found [0.0]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.failNotEquals(Assert.java:969)
at org.testng.Assert.assertEquals(Assert.java:640)
at com.epam.cdp.tests.CosCalTest.CosTest(CosCalTest.java:11)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1175)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at com.epam.cdp.tests.testng.Runner.main(Runner.java:24)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- CosTest -->
        <test-method status="FAIL" signature="CosTest(double, double)[pri:0, instance:com.epam.cdp.tests.CosCalTest@1b6a447]" name="CosTest" duration-ms="0" started-at="2019-08-19T15:51:39 GMT+04:00" data-provider="valuesForCosTest" finished-at="2019-08-19T15:51:39 GMT+04:00">
          <params>
            <param index="0">
              <value>
                <![CDATA[180]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[-1]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Invalid result of cos operation! did not expect to find [-1.0] but found [-0.8011526357338304]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Invalid result of cos operation! did not expect to find [-1.0] but found [-0.8011526357338304]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.failNotEquals(Assert.java:969)
at org.testng.Assert.assertEquals(Assert.java:640)
at com.epam.cdp.tests.CosCalTest.CosTest(CosCalTest.java:11)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1175)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at com.epam.cdp.tests.testng.Runner.main(Runner.java:24)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- CosTest -->
        <test-method status="FAIL" signature="CosTest(double, double)[pri:0, instance:com.epam.cdp.tests.CosCalTest@1b6a447]" name="CosTest" duration-ms="0" started-at="2019-08-19T15:51:39 GMT+04:00" data-provider="valuesForCosTest" finished-at="2019-08-19T15:51:39 GMT+04:00">
          <params>
            <param index="0">
              <value>
                <![CDATA[60]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[0.5]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Invalid result of cos operation! did not expect to find [0.5] but found [-0.3048106211022167]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Invalid result of cos operation! did not expect to find [0.5] but found [-0.3048106211022167]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.failNotEquals(Assert.java:969)
at org.testng.Assert.assertEquals(Assert.java:640)
at com.epam.cdp.tests.CosCalTest.CosTest(CosCalTest.java:11)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1175)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at com.epam.cdp.tests.testng.Runner.main(Runner.java:24)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- CosTest -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.epam.cdp.tests.CosCalTest@1b6a447]" name="setUp" is-config="true" duration-ms="0" started-at="2019-08-19T15:51:39 GMT+04:00" finished-at="2019-08-19T15:51:39 GMT+04:00">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
      </class> <!-- com.epam.cdp.tests.CosCalTest -->
    </test> <!-- Cos Cal Test -->
    <test name="Ctg Cal Test" duration-ms="0" started-at="2019-08-19T15:51:39 GMT+04:00" finished-at="2019-08-19T15:51:39 GMT+04:00">
      <class name="com.epam.cdp.tests.CtgCalTest">
        <test-method status="FAIL" signature="TgTest(double, double)[pri:0, instance:com.epam.cdp.tests.CtgCalTest@181a1bc]" name="TgTest" duration-ms="0" started-at="2019-08-19T15:51:39 GMT+04:00" data-provider="valuesForCtgTest" finished-at="2019-08-19T15:51:39 GMT+04:00">
          <params>
            <param index="0">
              <value>
                <![CDATA[90]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[0]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Invalid result of ctg operation! did not expect to find [0.0] but found [1.0]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Invalid result of ctg operation! did not expect to find [0.0] but found [1.0]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.failNotEquals(Assert.java:969)
at org.testng.Assert.assertEquals(Assert.java:640)
at com.epam.cdp.tests.CtgCalTest.TgTest(CtgCalTest.java:11)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1175)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at com.epam.cdp.tests.testng.Runner.main(Runner.java:24)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TgTest -->
        <test-method status="FAIL" signature="TgTest(double, double)[pri:0, instance:com.epam.cdp.tests.CtgCalTest@181a1bc]" name="TgTest" duration-ms="0" started-at="2019-08-19T15:51:39 GMT+04:00" data-provider="valuesForCtgTest" finished-at="2019-08-19T15:51:39 GMT+04:00">
          <params>
            <param index="0">
              <value>
                <![CDATA[270]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[0]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Invalid result of ctg operation! did not expect to find [0.0] but found [1.0]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Invalid result of ctg operation! did not expect to find [0.0] but found [1.0]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.failNotEquals(Assert.java:969)
at org.testng.Assert.assertEquals(Assert.java:640)
at com.epam.cdp.tests.CtgCalTest.TgTest(CtgCalTest.java:11)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1175)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at com.epam.cdp.tests.testng.Runner.main(Runner.java:24)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TgTest -->
        <test-method status="PASS" signature="TgTest(double, double)[pri:0, instance:com.epam.cdp.tests.CtgCalTest@181a1bc]" name="TgTest" duration-ms="0" started-at="2019-08-19T15:51:39 GMT+04:00" data-provider="valuesForCtgTest" finished-at="2019-08-19T15:51:39 GMT+04:00">
          <params>
            <param index="0">
              <value>
                <![CDATA[45]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[1]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TgTest -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.epam.cdp.tests.CtgCalTest@181a1bc]" name="tearDown" is-config="true" duration-ms="0" started-at="2019-08-19T15:51:39 GMT+04:00" finished-at="2019-08-19T15:51:39 GMT+04:00">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.epam.cdp.tests.CtgCalTest@181a1bc]" name="setUp" is-config="true" duration-ms="0" started-at="2019-08-19T15:51:39 GMT+04:00" finished-at="2019-08-19T15:51:39 GMT+04:00">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
      </class> <!-- com.epam.cdp.tests.CtgCalTest -->
    </test> <!-- Ctg Cal Test -->
    <test name="IsNegative Cal Test" duration-ms="15" started-at="2019-08-19T15:51:40 GMT+04:00" finished-at="2019-08-19T15:51:40 GMT+04:00">
      <class name="com.epam.cdp.tests.IsNegativeCalTest">
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.epam.cdp.tests.IsNegativeCalTest@bc464]" name="setUp" is-config="true" duration-ms="15" started-at="2019-08-19T15:51:40 GMT+04:00" finished-at="2019-08-19T15:51:40 GMT+04:00">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.epam.cdp.tests.IsNegativeCalTest@bc464]" name="tearDown" is-config="true" duration-ms="0" started-at="2019-08-19T15:51:40 GMT+04:00" finished-at="2019-08-19T15:51:40 GMT+04:00">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="IsNegativeTest(long, boolean)[pri:0, instance:com.epam.cdp.tests.IsNegativeCalTest@bc464]" name="IsNegativeTest" duration-ms="0" started-at="2019-08-19T15:51:40 GMT+04:00" data-provider="valuesForIsNegativeTest" finished-at="2019-08-19T15:51:40 GMT+04:00">
          <params>
            <param index="0">
              <value>
                <![CDATA[0]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[false]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- IsNegativeTest -->
        <test-method status="PASS" signature="IsNegativeTest(long, boolean)[pri:0, instance:com.epam.cdp.tests.IsNegativeCalTest@bc464]" name="IsNegativeTest" duration-ms="0" started-at="2019-08-19T15:51:40 GMT+04:00" data-provider="valuesForIsNegativeTest" finished-at="2019-08-19T15:51:40 GMT+04:00">
          <params>
            <param index="0">
              <value>
                <![CDATA[1]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[false]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- IsNegativeTest -->
        <test-method status="PASS" signature="IsNegativeTest(long, boolean)[pri:0, instance:com.epam.cdp.tests.IsNegativeCalTest@bc464]" name="IsNegativeTest" duration-ms="0" started-at="2019-08-19T15:51:40 GMT+04:00" data-provider="valuesForIsNegativeTest" finished-at="2019-08-19T15:51:40 GMT+04:00">
          <params>
            <param index="0">
              <value>
                <![CDATA[90]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[false]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- IsNegativeTest -->
        <test-method status="PASS" signature="IsNegativeTest(long, boolean)[pri:0, instance:com.epam.cdp.tests.IsNegativeCalTest@bc464]" name="IsNegativeTest" duration-ms="0" started-at="2019-08-19T15:51:40 GMT+04:00" data-provider="valuesForIsNegativeTest" finished-at="2019-08-19T15:51:40 GMT+04:00">
          <params>
            <param index="0">
              <value>
                <![CDATA[-2]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[true]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- IsNegativeTest -->
      </class> <!-- com.epam.cdp.tests.IsNegativeCalTest -->
    </test> <!-- IsNegative Cal Test -->
    <test name="IsPositive Cal Test" duration-ms="0" started-at="2019-08-19T15:51:40 GMT+04:00" finished-at="2019-08-19T15:51:40 GMT+04:00">
      <class name="com.epam.cdp.tests.IsPositiveCalTest">
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.epam.cdp.tests.IsPositiveCalTest@1624da6]" name="setUp" is-config="true" duration-ms="0" started-at="2019-08-19T15:51:40 GMT+04:00" finished-at="2019-08-19T15:51:40 GMT+04:00">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="IsPositiveTest(long, boolean)[pri:0, instance:com.epam.cdp.tests.IsPositiveCalTest@1624da6]" name="IsPositiveTest" duration-ms="0" started-at="2019-08-19T15:51:40 GMT+04:00" data-provider="valuesForIsPositiveTest" finished-at="2019-08-19T15:51:40 GMT+04:00">
          <params>
            <param index="0">
              <value>
                <![CDATA[0]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[false]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- IsPositiveTest -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.epam.cdp.tests.IsPositiveCalTest@1624da6]" name="tearDown" is-config="true" duration-ms="0" started-at="2019-08-19T15:51:40 GMT+04:00" finished-at="2019-08-19T15:51:40 GMT+04:00">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="IsPositiveTest(long, boolean)[pri:0, instance:com.epam.cdp.tests.IsPositiveCalTest@1624da6]" name="IsPositiveTest" duration-ms="0" started-at="2019-08-19T15:51:40 GMT+04:00" data-provider="valuesForIsPositiveTest" finished-at="2019-08-19T15:51:40 GMT+04:00">
          <params>
            <param index="0">
              <value>
                <![CDATA[90]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[true]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- IsPositiveTest -->
        <test-method status="PASS" signature="IsPositiveTest(long, boolean)[pri:0, instance:com.epam.cdp.tests.IsPositiveCalTest@1624da6]" name="IsPositiveTest" duration-ms="0" started-at="2019-08-19T15:51:40 GMT+04:00" data-provider="valuesForIsPositiveTest" finished-at="2019-08-19T15:51:40 GMT+04:00">
          <params>
            <param index="0">
              <value>
                <![CDATA[1]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[true]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- IsPositiveTest -->
        <test-method status="PASS" signature="IsPositiveTest(long, boolean)[pri:0, instance:com.epam.cdp.tests.IsPositiveCalTest@1624da6]" name="IsPositiveTest" duration-ms="0" started-at="2019-08-19T15:51:40 GMT+04:00" data-provider="valuesForIsPositiveTest" finished-at="2019-08-19T15:51:40 GMT+04:00">
          <params>
            <param index="0">
              <value>
                <![CDATA[-2]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[false]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- IsPositiveTest -->
      </class> <!-- com.epam.cdp.tests.IsPositiveCalTest -->
    </test> <!-- IsPositive Cal Test -->
    <test name="Pow Cal Test" duration-ms="0" started-at="2019-08-19T15:51:40 GMT+04:00" finished-at="2019-08-19T15:51:40 GMT+04:00">
      <class name="com.epam.cdp.tests.PowCalTest">
        <test-method status="FAIL" signature="PowTest(double, double, double)[pri:0, instance:com.epam.cdp.tests.PowCalTest@1b52d12]" name="PowTest" duration-ms="0" started-at="2019-08-19T15:51:40 GMT+04:00" data-provider="valuesForPowTest" finished-at="2019-08-19T15:51:40 GMT+04:00">
          <params>
            <param index="0">
              <value>
                <![CDATA[10]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[23]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[33]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Invalid result of pow operation! did not expect to find [33.0] but found [1.0000000000000001E23]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Invalid result of pow operation! did not expect to find [33.0] but found [1.0000000000000001E23]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.failNotEquals(Assert.java:969)
at org.testng.Assert.assertEquals(Assert.java:640)
at com.epam.cdp.tests.PowCalTest.PowTest(PowCalTest.java:11)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1175)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at com.epam.cdp.tests.testng.Runner.main(Runner.java:24)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- PowTest -->
        <test-method status="FAIL" signature="PowTest(double, double, double)[pri:0, instance:com.epam.cdp.tests.PowCalTest@1b52d12]" name="PowTest" duration-ms="0" started-at="2019-08-19T15:51:40 GMT+04:00" data-provider="valuesForPowTest" finished-at="2019-08-19T15:51:40 GMT+04:00">
          <params>
            <param index="0">
              <value>
                <![CDATA[10]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[-1]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[9]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Invalid result of pow operation! did not expect to find [9.0] but found [0.1]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Invalid result of pow operation! did not expect to find [9.0] but found [0.1]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.failNotEquals(Assert.java:969)
at org.testng.Assert.assertEquals(Assert.java:640)
at com.epam.cdp.tests.PowCalTest.PowTest(PowCalTest.java:11)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1175)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at com.epam.cdp.tests.testng.Runner.main(Runner.java:24)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- PowTest -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.epam.cdp.tests.PowCalTest@1b52d12]" name="setUp" is-config="true" duration-ms="0" started-at="2019-08-19T15:51:40 GMT+04:00" finished-at="2019-08-19T15:51:40 GMT+04:00">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.epam.cdp.tests.PowCalTest@1b52d12]" name="tearDown" is-config="true" duration-ms="0" started-at="2019-08-19T15:51:40 GMT+04:00" finished-at="2019-08-19T15:51:40 GMT+04:00">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="FAIL" signature="PowTest(double, double, double)[pri:0, instance:com.epam.cdp.tests.PowCalTest@1b52d12]" name="PowTest" duration-ms="0" started-at="2019-08-19T15:51:40 GMT+04:00" data-provider="valuesForPowTest" finished-at="2019-08-19T15:51:40 GMT+04:00">
          <params>
            <param index="0">
              <value>
                <![CDATA[0]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[0]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[0]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Invalid result of pow operation! did not expect to find [0.0] but found [1.0]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Invalid result of pow operation! did not expect to find [0.0] but found [1.0]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.failNotEquals(Assert.java:969)
at org.testng.Assert.assertEquals(Assert.java:640)
at com.epam.cdp.tests.PowCalTest.PowTest(PowCalTest.java:11)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1175)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at com.epam.cdp.tests.testng.Runner.main(Runner.java:24)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- PowTest -->
        <test-method status="FAIL" signature="PowTest(double, double, double)[pri:0, instance:com.epam.cdp.tests.PowCalTest@1b52d12]" name="PowTest" duration-ms="0" started-at="2019-08-19T15:51:40 GMT+04:00" data-provider="valuesForPowTest" finished-at="2019-08-19T15:51:40 GMT+04:00">
          <params>
            <param index="0">
              <value>
                <![CDATA[-1]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[10]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[9]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Invalid result of pow operation! did not expect to find [9.0] but found [1.0]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Invalid result of pow operation! did not expect to find [9.0] but found [1.0]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.failNotEquals(Assert.java:969)
at org.testng.Assert.assertEquals(Assert.java:640)
at com.epam.cdp.tests.PowCalTest.PowTest(PowCalTest.java:11)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1175)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at com.epam.cdp.tests.testng.Runner.main(Runner.java:24)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- PowTest -->
      </class> <!-- com.epam.cdp.tests.PowCalTest -->
    </test> <!-- Pow Cal Test -->
    <test name="Sin Cal Test" duration-ms="0" started-at="2019-08-19T15:51:40 GMT+04:00" finished-at="2019-08-19T15:51:40 GMT+04:00">
      <class name="com.epam.cdp.tests.SinCalTest">
        <test-method status="FAIL" signature="SinTest(double, double)[pri:0, instance:com.epam.cdp.tests.SinCalTest@10f6bfd]" name="SinTest" duration-ms="0" started-at="2019-08-19T15:51:40 GMT+04:00" data-provider="valuesForSinTest" finished-at="2019-08-19T15:51:40 GMT+04:00">
          <params>
            <param index="0">
              <value>
                <![CDATA[270]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[-1]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Invalid result of sin operation! did not expect to find [-1.0] but found [-0.1760459464712114]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Invalid result of sin operation! did not expect to find [-1.0] but found [-0.1760459464712114]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.failNotEquals(Assert.java:969)
at org.testng.Assert.assertEquals(Assert.java:640)
at com.epam.cdp.tests.SinCalTest.SinTest(SinCalTest.java:11)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1175)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at com.epam.cdp.tests.testng.Runner.main(Runner.java:24)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SinTest -->
        <test-method status="FAIL" signature="SinTest(double, double)[pri:0, instance:com.epam.cdp.tests.SinCalTest@10f6bfd]" name="SinTest" duration-ms="0" started-at="2019-08-19T15:51:40 GMT+04:00" data-provider="valuesForSinTest" finished-at="2019-08-19T15:51:40 GMT+04:00">
          <params>
            <param index="0">
              <value>
                <![CDATA[30]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[0.5]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Invalid result of sin operation! did not expect to find [0.5] but found [-0.9880316240928618]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Invalid result of sin operation! did not expect to find [0.5] but found [-0.9880316240928618]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.failNotEquals(Assert.java:969)
at org.testng.Assert.assertEquals(Assert.java:640)
at com.epam.cdp.tests.SinCalTest.SinTest(SinCalTest.java:11)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1175)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at com.epam.cdp.tests.testng.Runner.main(Runner.java:24)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SinTest -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.epam.cdp.tests.SinCalTest@10f6bfd]" name="setUp" is-config="true" duration-ms="0" started-at="2019-08-19T15:51:40 GMT+04:00" finished-at="2019-08-19T15:51:40 GMT+04:00">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.epam.cdp.tests.SinCalTest@10f6bfd]" name="tearDown" is-config="true" duration-ms="0" started-at="2019-08-19T15:51:40 GMT+04:00" finished-at="2019-08-19T15:51:40 GMT+04:00">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="FAIL" signature="SinTest(double, double)[pri:0, instance:com.epam.cdp.tests.SinCalTest@10f6bfd]" name="SinTest" duration-ms="0" started-at="2019-08-19T15:51:40 GMT+04:00" data-provider="valuesForSinTest" finished-at="2019-08-19T15:51:40 GMT+04:00">
          <params>
            <param index="0">
              <value>
                <![CDATA[90]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[1]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Invalid result of sin operation! did not expect to find [1.0] but found [0.8939966636005579]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Invalid result of sin operation! did not expect to find [1.0] but found [0.8939966636005579]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.failNotEquals(Assert.java:969)
at org.testng.Assert.assertEquals(Assert.java:640)
at com.epam.cdp.tests.SinCalTest.SinTest(SinCalTest.java:11)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1175)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at com.epam.cdp.tests.testng.Runner.main(Runner.java:24)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SinTest -->
        <test-method status="PASS" signature="SinTest(double, double)[pri:0, instance:com.epam.cdp.tests.SinCalTest@10f6bfd]" name="SinTest" duration-ms="0" started-at="2019-08-19T15:51:40 GMT+04:00" data-provider="valuesForSinTest" finished-at="2019-08-19T15:51:40 GMT+04:00">
          <params>
            <param index="0">
              <value>
                <![CDATA[0]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[0]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SinTest -->
      </class> <!-- com.epam.cdp.tests.SinCalTest -->
    </test> <!-- Sin Cal Test -->
    <test name="Sqrt Cal Test" duration-ms="0" started-at="2019-08-19T15:51:40 GMT+04:00" finished-at="2019-08-19T15:51:40 GMT+04:00">
      <class name="com.epam.cdp.tests.SqrtCalTest">
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.epam.cdp.tests.SqrtCalTest@1a03464]" name="setUp" is-config="true" duration-ms="0" started-at="2019-08-19T15:51:40 GMT+04:00" finished-at="2019-08-19T15:51:40 GMT+04:00">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="SqrtTest(double, double)[pri:0, instance:com.epam.cdp.tests.SqrtCalTest@1a03464]" name="SqrtTest" duration-ms="0" started-at="2019-08-19T15:51:40 GMT+04:00" data-provider="valuesForSqrtTest" finished-at="2019-08-19T15:51:40 GMT+04:00">
          <params>
            <param index="0">
              <value>
                <![CDATA[4]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[2]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SqrtTest -->
        <test-method status="PASS" signature="SqrtTest(double, double)[pri:0, instance:com.epam.cdp.tests.SqrtCalTest@1a03464]" name="SqrtTest" duration-ms="0" started-at="2019-08-19T15:51:40 GMT+04:00" data-provider="valuesForSqrtTest" finished-at="2019-08-19T15:51:40 GMT+04:00">
          <params>
            <param index="0">
              <value>
                <![CDATA[16]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[4]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SqrtTest -->
        <test-method status="PASS" signature="SqrtTest(double, double)[pri:0, instance:com.epam.cdp.tests.SqrtCalTest@1a03464]" name="SqrtTest" duration-ms="0" started-at="2019-08-19T15:51:40 GMT+04:00" data-provider="valuesForSqrtTest" finished-at="2019-08-19T15:51:40 GMT+04:00">
          <params>
            <param index="0">
              <value>
                <![CDATA[9]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[3]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SqrtTest -->
        <test-method status="PASS" signature="SqrtTest(double, double)[pri:0, instance:com.epam.cdp.tests.SqrtCalTest@1a03464]" name="SqrtTest" duration-ms="0" started-at="2019-08-19T15:51:40 GMT+04:00" data-provider="valuesForSqrtTest" finished-at="2019-08-19T15:51:40 GMT+04:00">
          <params>
            <param index="0">
              <value>
                <![CDATA[100]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[10]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SqrtTest -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.epam.cdp.tests.SqrtCalTest@1a03464]" name="tearDown" is-config="true" duration-ms="0" started-at="2019-08-19T15:51:40 GMT+04:00" finished-at="2019-08-19T15:51:40 GMT+04:00">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.epam.cdp.tests.SqrtCalTest -->
    </test> <!-- Sqrt Cal Test -->
    <test name="Tg Cal Test" duration-ms="0" started-at="2019-08-19T15:51:41 GMT+04:00" finished-at="2019-08-19T15:51:41 GMT+04:00">
      <class name="com.epam.cdp.tests.TgCalTest">
        <test-method status="FAIL" signature="TgTest(double, double)[pri:0, instance:com.epam.cdp.tests.TgCalTest@1f97b55]" name="TgTest" duration-ms="0" started-at="2019-08-19T15:51:41 GMT+04:00" data-provider="valuesForTgTest" finished-at="2019-08-19T15:51:41 GMT+04:00">
          <params>
            <param index="0">
              <value>
                <![CDATA[360]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[0]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Invalid result of tg operation! did not expect to find [0.0] but found [1.0]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Invalid result of tg operation! did not expect to find [0.0] but found [1.0]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.failNotEquals(Assert.java:969)
at org.testng.Assert.assertEquals(Assert.java:640)
at com.epam.cdp.tests.TgCalTest.TgTest(TgCalTest.java:11)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1175)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at com.epam.cdp.tests.testng.Runner.main(Runner.java:24)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TgTest -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.epam.cdp.tests.TgCalTest@1f97b55]" name="setUp" is-config="true" duration-ms="0" started-at="2019-08-19T15:51:41 GMT+04:00" finished-at="2019-08-19T15:51:41 GMT+04:00">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="TgTest(double, double)[pri:0, instance:com.epam.cdp.tests.TgCalTest@1f97b55]" name="TgTest" duration-ms="0" started-at="2019-08-19T15:51:41 GMT+04:00" data-provider="valuesForTgTest" finished-at="2019-08-19T15:51:41 GMT+04:00">
          <params>
            <param index="0">
              <value>
                <![CDATA[45]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[1]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TgTest -->
        <test-method status="FAIL" signature="TgTest(double, double)[pri:0, instance:com.epam.cdp.tests.TgCalTest@1f97b55]" name="TgTest" duration-ms="0" started-at="2019-08-19T15:51:41 GMT+04:00" data-provider="valuesForTgTest" finished-at="2019-08-19T15:51:41 GMT+04:00">
          <params>
            <param index="0">
              <value>
                <![CDATA[0]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[0]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Invalid result of tg operation! did not expect to find [0.0] but found [NaN]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Invalid result of tg operation! did not expect to find [0.0] but found [NaN]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.failNotEquals(Assert.java:969)
at org.testng.Assert.assertEquals(Assert.java:640)
at com.epam.cdp.tests.TgCalTest.TgTest(TgCalTest.java:11)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1175)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at com.epam.cdp.tests.testng.Runner.main(Runner.java:24)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TgTest -->
        <test-method status="FAIL" signature="TgTest(double, double)[pri:0, instance:com.epam.cdp.tests.TgCalTest@1f97b55]" name="TgTest" duration-ms="0" started-at="2019-08-19T15:51:41 GMT+04:00" data-provider="valuesForTgTest" finished-at="2019-08-19T15:51:41 GMT+04:00">
          <params>
            <param index="0">
              <value>
                <![CDATA[180]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[0]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Invalid result of tg operation! did not expect to find [0.0] but found [1.0]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Invalid result of tg operation! did not expect to find [0.0] but found [1.0]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.failNotEquals(Assert.java:969)
at org.testng.Assert.assertEquals(Assert.java:640)
at com.epam.cdp.tests.TgCalTest.TgTest(TgCalTest.java:11)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1175)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at com.epam.cdp.tests.testng.Runner.main(Runner.java:24)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TgTest -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.epam.cdp.tests.TgCalTest@1f97b55]" name="tearDown" is-config="true" duration-ms="0" started-at="2019-08-19T15:51:41 GMT+04:00" finished-at="2019-08-19T15:51:41 GMT+04:00">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.epam.cdp.tests.TgCalTest -->
    </test> <!-- Tg Cal Test -->
  </suite> <!-- Demo Suite -->
  <suite name="Default Suite" duration-ms="0" started-at="2019-08-19T15:51:41 GMT+04:00" finished-at="2019-08-19T15:51:41 GMT+04:00">
    <groups>
    </groups>
  </suite> <!-- Default Suite -->
</testng-results>
